test/integration/singlecluster/controller/admissionchecks/provisioning/provisioning_test.go

```diff
@@ -666,6 +667,7 @@ var _ = ginkgo.Describe("Provisioning", ginkgo.Ordered, ginkgo.ContinueOnFailure
 			})
 
 			ginkgo.By("Setting the provisioning request as BookingExpired", func() {
+				time.Sleep(100 * time.Millisecond)
```

Comment:
Maybe let's add a comment
// wait a bit to make it almost certain that the provisioning controller already sees the workload as admitted.

Reply:
Why not just using `LongTimeout`?

Reply:
This is not a timeout, we just want to give the provisioning controller a bit more time.

Reply:
Uhm, that makes sense. Let's just add sleep since this is integration tests.
We want to avoid the checking internal cache as much as possible.

Reply:
Done